packaging/docker/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+version: "3"
+
+services:
+  clef:
+    image: ethersphere/clef:0.4.2
+    restart: unless-stopped
+    environment:
+      - CLEF_CHAINID
+    volumes:
+      - clef:/app/data
+    command: full
+
+  bee:
+    image: ethersphere/bee:0.4.0
```

Comment:
We will forget to update version at some point.

Maybe we should add make step to update on all places?

Reply:
I agree with you but not sure how to set that up